velox/connectors/tpch/TpchConnector.h

```diff
@@ -75,7 +75,8 @@ class TpchDataSource : public DataSource {
       const std::unordered_map<
           std::string,
           std::shared_ptr<connector::ColumnHandle>>& columnHandles,
-      velox::memory::MemoryPool* FOLLY_NONNULL pool);
+      velox::memory::MemoryPool* FOLLY_NONNULL pool,
+      const std::shared_ptr<dwio::common::ReaderOptions>& options = {});
```

Comment:
Remove the `pool` since you added options or remove options.
Remove `shared_ptr<>` as well to be consistent with other places.

Reply:
Ok, Done. Waiting on my UT to complete before I push.